scripts/commands/updateApiDocs.ts

```diff
@@ -91,7 +97,20 @@ zxMain(async () => {
     );
   }
 
-  const type = args.historical ? "historical" : "latest";
+  if (args.historical && args.dev) {
+    throw new Error(
+      `${args.package} ${args.version} cannot be historical and dev at the same time. Please remove at least only one of these two arguments: --historical, --dev.`,
+    );
+  }
+
+  const devRegex = /[0-9](-rc|-dev)/;
+  if (args.dev && !args.version.match(devRegex)) {
+    throw new Error(
+      `${args.package} ${args.version} is not a correct dev version. Please make sure the version has one of the following suffixes with a hyphen immediately following the patch version: -rc, -dev. e.g. 1.0.0-rc1 or 1.0.0-dev`,
```

Comment:
Following the semver specification we need to have a hyphen between the patch version and the prerelease identifiers. I tested it and without it, we won't be able to sort the versions without adding extra logic on the version sector code.

For more information: https://semver.org/#spec-item-9

Reply:
For rc1, there is no `-`, e.g. `1.0.0rc1`. See https://pypi.org/project/qiskit/#history. We need to use the version from the actual release because that's what users have to put into their Python install.

So we'll need to adapt the internal code to work around this.

Reply:
Thanks for pointing that out! I changed the script to accept release candidate versions without the dash, but this is something super important to make sure we correct when we read the version from the `_package.json` on iqp-channel-docs because otherwise, it will break components that are not able to recognize it